internal/commands/sbomtest/presenter.go

```diff
@@ -127,11 +127,19 @@ func RenderPrettyResult(w io.Writer, orgID, filepath string, res *snykclient.SBO
 	untested := make([]view.Component, 0, len(res.Summary.Untested))
 
 	for i := range res.Vulnerabilities {
+		introducedBy := make([]view.IntroducedBy, 0, len(res.Vulnerabilities[i].Packages))
+		for _, pkg := range res.Vulnerabilities[i].Packages {
+			introducedBy = append(introducedBy, view.IntroducedBy{
+				Name:    pkg.Name,
+				Version: pkg.Version,
+				PURL:    pkg.PURL,
+			})
+		}
```

Comment:
I would add a new line inbetween the for loop and the issues assignments, since they are building up two different things